async_upnp_client/ssdp.py

```diff
@@ -66,9 +121,15 @@ def decode_ssdp_packet(data: bytes, addr: str) -> Tuple[str, CaseInsensitiveDict
     email_headers = email.message_from_bytes(header_lines)
     headers = CaseInsensitiveDict(**dict(email_headers.items()))
 
+    # adjust some headers
+    if "location" in headers:
+        headers["location"] = get_adjusted_url(headers["location"], addr)
+
     # own data
     headers["_timestamp"] = datetime.now()
-    headers["_address"] = addr
+    headers["_host"] = get_host_string(addr)
+    headers["_port"] = addr[1]
```

Comment:
Could theoretically be considered a breaking change since the _address field goes away. I need the separate host part to compare on a directed search. The reply is quite often from a different port, which per say is broken (or at least breaks NAT traffic), but common.